### PR TITLE
Release 1.1.13

### PR DESCRIPTION
**Pull request by lifubang**
## [1.1.13] - 2024-06-13

> There is no certainty in the world. This is the only certainty I have.

### Important Notes
 * If building with Go 1.22.x, make sure to use 1.22.4 or a later version.
   (see #4233 for more details)

### Fixed

 * Support go 1.22.4+. (#4313)
 * runc list: fix race with runc delete. (#4231)
 * Fix set nofile rlimit error. (#4277, #4299)
 * libct/cg/fs: fix setting rt_period vs rt_runtime. (#4284)
 * Fix a debug msg for user ns in nsexec. (#4315)
 * script/*: fix gpg usage wrt keyboxd. (#4316)
 * CI fixes and misc backports. (#4241)
 * Fix codespell warnings. (#4300)

### Changed

 * Silence security false positives from golang/net. (#4244)
 * libcontainer: allow containers to make apps think fips is enabled/disabled for testing. (#4257)
 * allow overriding VERSION value in Makefile. (#4270)
 * Vagrantfile.fedora: bump Fedora to 39. (#4261)
 * ci/cirrus: rm centos stream 8. (#4305, #4308)

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

**Review by rata — APPROVED** [review]
This LGTM, but other reviewers pelase note that there are two PRs referenced not yet merged. We need to merge those first

File: CHANGELOG.md [hunk 1]
```diff
@@ -511,7 +538,8 @@ implementation (libcontainer) is *not* covered by this policy.
 [1.0.1]: https://github.com/opencontainers/runc/compare/v1.0.0...v1.0.1
 
 <!-- 1.1.z patch releases -->
-[Unreleased 1.1.z]: https://github.com/opencontainers/runc/compare/v1.1.12...release-1.1
+[Unreleased 1.1.z]: https://github.com/opencontainers/runc/compare/v1.1.13...release-1.1
+[1.1.13]: https://github.com/opencontainers/runc/compare/v1.1.11...v1.1.13
```
Comment (kolyshkin) [inline CHANGELOG.md:542]:
```suggestion
[1.1.13]: https://github.com/opencontainers/runc/compare/v1.1.12...v1.1.13
```

**Comment by kolyshkin** [comment]
> This LGTM, but other reviewers pelase note that there are two PRs referenced not yet merged. We need to merge those first

Basically we're looking into the relevant milestone (https://github.com/opencontainers/runc/milestone/28 in this case) as well as backport/todo/1.1 PRs (including the closed ones!)

**Comment by kolyshkin** [comment]
@lifubang can you rebase this? Has to be done from the command line because we want these commits to be signed.

**Comment by kolyshkin** [comment]
@cyphar PTAL

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased 1.1.z]
 
+## [1.1.13] - 2024-06-11
```
Comment (lifubang) [inline CHANGELOG.md:9]: Change to `2024-06-12`?

**Comment by cyphar** [comment]
@lifubang Can you please PGP-sign these commits?

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+ * Fix codespell warnings. (#4300)
+ * Fix a debug msg for user ns in nsexec. (#4315)
+ * script/*: fix gpg usage wrt keyboxd. (#4316)
+
+### Changed
+
+ * CI fixes and misc backports. (#4241)
+ * Silence security false positives from golang/net. (#4244)
+ * Vagrantfile.fedora: bump Fedora to 39. (#4261)
+ * libcontainer: force apps to think fips is enabled/disabled for testing. (#4257)
```
Comment (cyphar) [inline CHANGELOG.md:32]: The commit message for this change didn't make sense, this is a better way of phrasing what the change actually was:

```suggestion
 * libcontainer: allow containers to make apps think fips is enabled/disabled for testing. (#4257)
```

**Review by cyphar — COMMENTED** [review]
Minor nit.

**Comment by lifubang** [comment]
> Can you please PGP-sign these commits?

Resigned with gpg.

**Review by cyphar — APPROVED** [review]
LGTM. @kolyshkin said he wanted to do the release this time, so he can merge it and all that jazz. Thanks!

**Review by kolyshkin — APPROVED** [review]
LGTM, thanks!

**Pull request merged**